breadbox/breadbox/schemas/dataset.py

```diff
@@ -192,6 +178,27 @@ def check_valid_value_type_with_allowed_values(self):
             )
         return self
 
+    @field_validator("allowed_values", mode="after")
+    @classmethod
+    def check_valid_allowed_values(cls, v: Optional[List[str]]):
+        """
+        Checks there are no empty strings and no repeated allowed values. Values in allowed values list are not case-sensitive.
+        """
+        if v is None:
+            return v
+        # Decision to make allowed values not case-sensitive in case user error in accidental repeats
```

Comment:
This is carried over from our past decisions but I've written the comment to make it more visible. Example of `"true"` and `True` should be considered the same value